schema/tables/npm_packages.yml

```
@@ -7,12 +7,9 @@ columns:
     platforms:
       - linux
 examples: >-
-  List the author, description and more information about packages made by Fleet. Replace the
-  homepage with any other distributor desired.
+  List the author, description and more information about the NPM package called `webpack`, if installed:
 
-  ```
-
-  SELECT author, description, directory, version FROM npm_packages WHERE homepage='https://fleetdm.com';
-  
+  ```sql
+  SELECT author, description, directory, version FROM npm_packages WHERE name='webpack';
   ```
 
```

Should we clarify that these are globally installed packages in a `description` override? e.g., 
```suggestion
description: Node packages globally installed on a system.
```

---

@eashaw Yeah, and probably also say "Node.js" (because "Node" can be generically understood to mean many different things by different people)